<ui version="4.0" >
 <class>SettingsJabberForm</class>
 <widget class="QWidget" name="SettingsJabberForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Jabber</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QCheckBox" name="checkEnable" >
     <property name="text" >
      <string>Enable remote control via Jabber</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Account credentials</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>JID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3" >
       <widget class="QLineEdit" name="lineJID" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3" >
       <widget class="QLineEdit" name="linePassword" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Priority</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="spinPriority" >
        <property name="minimum" >
         <number>-99</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Resource name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLineEdit" name="lineResource" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QCheckBox" name="checkAutoAuth" >
     <property name="text" >
      <string>Grant roster authorizations upon request</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QCheckBox" name="checkRestrictSelf" >
     <property name="text" >
      <string>Restrict access to my own JID</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupRestrictPassword" >
     <property name="title" >
      <string>Restrict access using a password</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLineEdit" name="lineRestrictPassword" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Proxy</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QComboBox" name="comboProxy" />
   </item>
   <item row="6" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../gfx/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>groupRestrictPassword</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineRestrictPassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>127</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel" >
     <x>152</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
